src/scripts/lib/server/utils/server-config/index.js

```diff
@@ -0,0 +1,26 @@
+const Configstore = require('configstore')
+const inq = require('inquirer')
+const prompts = require('./setup-prompts')
+
+module.exports = class ServerConfig {
+
+  constructor(defaults) {
+    this.db = new Configstore('wowser', defaults)
+  }
+
+  initSetup() {
```

Comment:
Perhaps renaming this to `prompt` would convey more meaning as to what it'll do?